app/src/main/AndroidManifest.xml

```diff
@@ -4,18 +4,22 @@
 
     <application
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/app_logo"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:largeHeap="true"
+        android:roundIcon="@drawable/app_logo"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name="com.globant.pokemontcgapp.MainActivity">
+        android:theme="@style/AppTheme.NoActionBar">
+        <activity
+            android:name="com.globant.pokemontcgapp.activity.SplashScreenActivity"
+            android:hardwareAccelerated="true">
```

Comment:
why do you need this line?

Reply:
that was an old fixed i used to help the animations run smoother, with Lottie i don't need it, thanks for the catch